src/numerical_flux/entropy_cons_numerical_flux.cpp

```diff
@@ -0,0 +1,70 @@
+#include "entropy_cons_numerical_flux.hpp"
+#include "ADTypes.hpp"
+
```

Comment:
consider moving the contents of this file into `convective_numerical_flux.cpp`

Reply:
fixed as per issue #125